backend/online-bookmark/api/swagger/swagger.yaml

```diff
@@ -192,7 +228,61 @@ paths:
         "200":
           description: A link
           schema:
-             $ref: "#/definitions/Link"
+            $ref: "#/definitions/Link"
+        "401":
+          description: Unathorized access
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+  /user:
+    x-swagger-router-controller: user
+    post:
+      operationId: createUser
+      summary: Creates a user
+      parameters:
+        - name: user
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/User"
+      responses:
+        "200":
+          description: User created
+        "409":
+          description: Username, or email address in use
```

Comment:
You say, the username or the email address is in use, yet I don't see "username" among the required fields. Am I not seeing something?